manifests/w/Workday,Inc/PlanningforExcel/2022.105.2718.2929/Workday,Inc.PlanningforExcel.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Workday,Inc.PlanningforExcel

```

```suggestion
PackageIdentifier: WorkdayInc.PlanningforExcel
```
Best to omit commas in package identifiers